.travis.yml

```diff
@@ -2,11 +2,8 @@ language: objective-c
 osx_image: xcode7.3
 xcode_sdk: 
     - iphonesimulator9.3
-    - iphonesimulator9.2
 xcode_project: RileyLink.xcodeproj
 xcode_scheme: 
     - RileyLink
 script:
-    # We need to enumerate these manually until xctool is fixed
     - xcodebuild -project RileyLink.xcodeproj -scheme RileyLink -sdk iphonesimulator9.3 test
-    - xcodebuild -project RileyLink.xcodeproj -scheme RileyLink -sdk iphonesimulator9.2 test
```

Comment:
@ps2 I'm not sure how to run these tests against the 9.2 simulator while still using xcode7.3 (Swift 2.2)